packages/node-core/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- When the `primary-network-endpoint` flag exists, it will report an "Invalid endpoint" error.(#2612)
```

Comment:
This doesn't clearly explain what was fixed, the issue wasn't related to this flag it was related to flags being overwritten and it would have been the same issue for any flags with special cases